acq4/devices/PMT/PMT.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from acq4.devices.OptomechDevice import OptomechDevice
+from acq4.devices.DAQGeneric import DAQGeneric
+
+class PMT(DAQGeneric, OptomechDevice):
+    def __init__(self, dm, config, name):
+        self.omConf = {}
+        for k in ['parentDevice', 'transform']:
+            if k in config:
+                self.omConf[k] = config.pop(k)
+        OptomechDevice.__init__(self, dm, config, name)
```

Comment:
`self.omConf` gives an error if placed instead of config:

`TypeError: string indices must be integers, not str`